tests/test_crs/valid_pgbench.yaml

```diff
@@ -17,6 +17,7 @@ spec:
       init_cmd_flags: ''
       scaling_factor: 1
       samples: 2
+      num_databases_pattern: 'all'
```

Comment:
not a blocker but highly suggest that for the testing we can include add_1 or add_2 patterns and test against multiple db if possible. This will increase coverage.

Reply:
+1

Reply:
I played with this briefly, but I think it will be better done in a later commit.